### PR TITLE
fixed quick start & hero issues

### LINKED ISSUE #318: quick-start bugs and some more issues
These issues:
https://app.clickup.com/t/2fz84fx
https://app.clickup.com/t/2fz86cv
https://app.clickup.com/t/2fz87fu

### PR DESCRIPTION
**Pull request by NitsanCohen770**
closed #318
most of the tasks done

### CODE REVIEW

File: bit/quick-start/learn-crossroad/learn-crossroad.module.scss [hunk 1]
```diff
@@ -34,6 +35,27 @@
     line-height: 24px;
     text-align: left;
   }
+
+  @media screen and (max-width: $br-md) {
+    .image {
+      justify-self: flex-start;
+      height: 100px;
+      width: 100px;
```
Comment (KutnerUri) [inline bit/quick-start/learn-crossroad/learn-crossroad.module.scss:43]: maybe just use relative size? `23%` or `10em`. This way you will not have to specify the size at each breakpoint
also, check the image dimensions above, the aspect ratio is different - 116x136 vs 100x100.
make sure the image is not distorted.

File: community/ui/hero/hero.module.scss [hunk 2]
```diff
@@ -114,6 +114,10 @@
   color: #ededed;
   width: 20em;
   height: 48px;
+  > div:nth-child(2) > span {
+    background-color: #fffbff;
+    color: #2b2b2b;
```
Comment (KutnerUri) [inline community/ui/hero/hero.module.scss:119]: use tokens. ⚠️

File: design/ui/cards/card/card.tsx [hunk 3]
```diff
@@ -9,10 +9,10 @@ export type CardProps = {
   children?: ReactNode;
 } & React.HTMLAttributes<HTMLDivElement>;
 
-export function Card({ children, className, ...rest }: CardProps) {
+export const Card = forwardRef<HTMLDivElement, CardProps>((props, ref) => {
```
Comment (KutnerUri) [inline design/ui/cards/card/card.tsx:12]: give a name to the inner function. The convention is `CardWithRef` :(

File: design/ui/cards/card/card.tsx [hunk 4]
```diff
@@ -9,10 +9,10 @@ export type CardProps = {
...
   return (
-    <div className={classNames(styles.card, className)} {...rest}>
-      {children}
+    <div ref={ref} className={classNames(styles.card, props.className)} {...props}>
+      {props.children}
```
Comment (KutnerUri) [inline design/ui/cards/card/card.tsx:15]: children are included in props, so you can skip them.

```suggestion
    <div ref={ref} className={classNames(styles.card, props.className)} {...props}/>
```

**Review by KutnerUri — CHANGES_REQUESTED** [review]

File: community/ui/hero/hero.module.scss [hunk 5]
```diff
@@ -109,11 +109,19 @@
...
   color: #ededed;
   width: 20em;
   height: 48px;
+  > div:nth-child(2) > span {
+    // @hack - override system tokens
+    --bit-text-on-accent: #2b2b2b;
+    --surface01-color: #fffbff;
+    background-color: var(--surface01-color);
+    color: var(--bit-text-on-accent);
+  }
```
Comment (KutnerUri) [inline community/ui/hero/hero.module.scss:124]:
```suggestion
  // @hack - override system tokens
  --bit-bg-dent: #0a1821;
  --bit-accent-color: #ededed;
  --bit-text-on-accent: #2b2b2b;
  color: #ededed;
  width: 20em;
  height: 48px;
```

Comment (KutnerUri) [inline community/ui/hero/hero.module.scss:124]:
```suggestion
  --bit-bg-dent: #0a1821;
  --bit-accent-color: #ededed;
  --bit-text-on-accent: #2b2b2b;
  color: #ededed;
  width: 20em;
  height: 48px;
```
```

File: docs/blocks/sidebar/tree/tree.tsx [hunk 6]
```diff
@@ -18,9 +18,19 @@ export function Tree({ displayTitle, tree, className, ...rest }: TreeProps) {
   const links = tree && tree.children;
   if (!links || links.length === 0) return null;
   const [active, setToActive] = useState(tree.id);
-
+  const ref = useRef<HTMLDivElement>(null);
+  useEffect(() => {
+    if (ref.current) {
+      ref.current.scrollIntoView({ behavior: 'auto', block: 'start' });
+    }
+  }, []);
```
Comment (KutnerUri) [inline docs/blocks/sidebar/tree/tree.tsx:26]: space line

**Review by KutnerUri — CHANGES_REQUESTED** [review]

File: bit/quick-start/card/card.mock.ts [hunk 7]
```diff
@@ -1,137 +1,134 @@
 export const cardsMock = [
   {
-    heading: "Wiki",
-    description:
-      "Collaborate on organizational process and knowledge with a component-driven wiki.",
+    heading: 'Wiki',
```
Comment (KutnerUri) [inline bit/quick-start/card/card.mock.ts:3]: STOP COMMITTING UNRELATED FILES

File: bit/quick-start/card/card.tsx [hunk 8]
```diff
@@ -56,7 +56,7 @@ export function Card({ heading, description, frameworkLogos, selected, children,
   };
   useEffect(() => {
     if (selected) {
-      cardRef.current?.scrollIntoView({ behavior: 'smooth', block: 'center' });
+      cardRef?.current?.scrollIntoView({ behavior: 'smooth', block: 'center' });
```
Comment (KutnerUri) [inline bit/quick-start/card/card.tsx:59]: unnecessary, remote

File: bit/quick-start/featured/featured.spec.tsx [hunk 9]
```diff
@@ -5,17 +5,20 @@ import { cardsMock } from './card.mock';
 
 describe('featured', () => {
   it('should render', () => {
+    window.HTMLElement.prototype.scrollIntoView = jest.fn();
```
Comment (KutnerUri) [inline bit/quick-start/featured/featured.spec.tsx:8]: move to beforeEach, and remove during cleanup

File: community/ui/graph/component-card-graph/component-card-node.tsx [hunk 10]
```diff
@@ -14,6 +14,7 @@ export type ComponentCardPayload = {
 
 export function ComponentCardNode({ node }: GraphNodeProps<ComponentCardPayload>) {
   const { preview, ...rest } = node.payload || {};
+  // @ts-ignore TODO fix this
```
Comment (KutnerUri) [inline community/ui/graph/component-card-graph/component-card-node.tsx:17]: maybe just update componentUrl / component id? it should match the bit version

**Review by KutnerUri — APPROVED** [review]

File: bit/content/what-is-bit/what-is-bit.module.scss [hunk 11]
```diff
@@ -1,14 +1,18 @@
-
+@import '~@teambit/base-ui.layout.breakpoints/_breakpoints.scss';
 .bitCard {
   float: left;
   max-width: 250px;
   margin: 30px;
+  @media screen and (max-width: $br-sm) {
+    float: none;
+    margin: auto;
+  }
```
[the remark this answers is not in the document]
Comment (ranm8) [reply bit/content/what-is-bit/what-is-bit.module.scss:9]: @NitsanCohen770

[the remark this answers is not in the document]
Comment (ranm8) [reply bit/content/what-is-bit/what-is-bit.module.scss:9]: @NitsanCohen770

**Comment by ranm8** [comment]
guys, can we get this PR done? @NitsanCohen770 @KutnerUri

File: community/component-showcase/component-showcase.module.scss [hunk 12]
```diff
@@ -65,7 +65,7 @@
   width: 100%;
   height: 100%;
   box-sizing: border-box;
-  padding: 8px 12px;
+  overflow-x: hidden;
```
Comment (KutnerUri) [inline community/component-showcase/component-showcase.module.scss:68]: add a comment explaining that this is for large previewed components

File: bit/content/what-is-bit/what-is-bit-qs.module.scss [hunk 13]
```diff
@@ -1,6 +0,0 @@
-.leftRight {
```
Comment (KutnerUri) [inline bit/content/what-is-bit/what-is-bit-qs.module.scss:1]: check this

**Review by KutnerUri — APPROVED** [review]

File: bit/content/what-is-bit/what-is-bit-qs.module.scss (hunk 13 above)
Comment (KutnerUri) [reply bit/content/what-is-bit/what-is-bit-qs.module.scss:1]: also delete the file?

**Pull request merged**